tests/flow/optional_props/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -187,7 +187,7 @@ b.b = 0; // because subsequent writes would widen the exact object
 (a.b: number); // error: property \`b\` not found
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 /* The logic that allows ({}: {p?:T}), described in test3.js, should _not_ also
-   fire for exact annotations. */
+ fire for exact annotations. */
```

Comment:
I think all changes are great, except this one. This one is a bit of a bug in master though:

**Prettier 1.14.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6AVAAmnAOlTQmAdwk3VXwyx30MxLIvxABoQIAHGAS2gGdkoAIYAnURGIAFMQkEphANwg8AJmxAAjUcLABrODADKnXTygBzZDFEBXOO3P84omFJ0WAtsOQAzYQA2zuwAVvwAHgBCOvqGRsKecAAy5nB+gcEgpqLOoshawpoAngHQGpyi5jAA6mowABbIABwADOwVEM7VOpz5FXC5imnsonAAjrY8o+7CXj5I-kEOIM6ePNZ2y-zmFgFwAIq2EPDpS+wwhbWqDcgATOc6PAE7AMIQnt75UDgats4AKoV5ItnABfUFAA)
```sh
--parser babylon
```

**Input:**
```jsx
/* one
   two */
/** one
   two */

```

**Output:**
```jsx
/* one
   two */
/** one
 two */

```

I think that the rule should be that every line has to _visually_ start with a `*`, including the `*` in the `/*` and `*/` markers themselves.

In the above cases the last line does not start with a star.

Reply:
Fixed:

**Prettier pr-5206**
[Playground link](https://deploy-preview-5206--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEB6AVAAmnAOlTQmAdwk3VXwyx30MxLIvxABoQIAHGAS2gGdkoAIYAnURGIAFMQkEphANwg8AJmxAAjUcLABrODADKnXTygBzZDFEBXOO3P84omFJ0WAtsOQAzYQA2zuwAVvwAHgBCOvqGRsKecAAy5nB+gcEgpqLOoshawpoAngHQGpyi5jAA6mowABbIABwADOwVEM7VOpz5FXC5imnsonAAjrY8o+7CXj5I-kEOIM6ePNZ2y-zmFgFwAIq2EPDpS+wwhbWqDcgATOc6PAE7AMIQnt75UDgats4AKoV5ItnABfUFAA)
```sh
--parser babylon
```

**Input:**
```jsx
/* one
   two */
/** one
   two */

```

**Output:**
```jsx
/* one
 two */
/** one
 two */

```